src/middlewares/mail.js

```diff
@@ -0,0 +1,40 @@
+/* eslint-disable import/prefer-default-export */
+import { verifyEmail } from '../services/mail/template/verifyEmail';
+import { emailVerifyToken } from '../utils/index';
+import Mail from '../services/mail/Mail';
+
+export const SendVerificationEmail = async (req, res, next) => {
+  let { email, firstName, lastName } = req.body;
+  email = email ? email.trim() : '';
+  firstName = firstName.trim();
+  lastName = lastName.trim();
+  const id = 'some_encoded_identiity';
+  const token = await emailVerifyToken(id);
+  const emaildDetails = {
+    Subject: 'Email Verification',
+    Recipient: email,
+  };
+  const link = `http://localhost:3000/api/v1/auth/verify?id=${token}`;
```

Comment:
this link is wrong. it shows you using `/auth/verify` but the app uses `/email/verify`. Also, I believe the endpoint should be `/users/email/test` and `/users/email/verify` since its more user-focused and it's in the users' route folder. Also, the `PORT` should be dynamic because a user might use a different `PORT` i.e use the `.env PORT`.

Reply:
Fixed